system_test/aest_hard_fork_SUITE.erl

```diff
@@ -445,6 +471,123 @@ new_node_can_mine_on_old_chain_using_new_protocol(Cfg) ->
     aest_nodes:kill_node(new_node3, Cfg),
     ok.
 
+%% New node can sync the old chain from other new node and can start mining
+%% on the top of the old chain. The new node can mine blocks using the old
+%% protocol and include spend transaction in the blocks.
+new_node_can_mine_spend_tx_on_old_chain_using_old_protocol(Cfg) ->
+    {_, TopHeight} = proplists:lookup(db_backup_top_height, Cfg),
+    {_, TopHash} = proplists:lookup(db_backup_top_hash, Cfg),
+    % Make sure the chain won't switch to the new protocol during the test.
+    HeightOfNewProtocol = TopHeight + 10000000,
+    aest_nodes:setup_nodes(
+      [?NEW_NODE1(HeightOfNewProtocol),
+       ?NEW_NODE2(HeightOfNewProtocol)], Cfg),
+    % Start new node and restore the old chain.
+    start_node(new_node1, Cfg),
+    {ok, {TopHash, TopHeight}} = restore_db_backup_on_node(new_node1, Cfg),
+    B1 = get_block_by_height(new_node1, TopHeight, Cfg),
+    ?assertEqual(TopHash, maps:get(hash, B1)),
+    %% Make sure balance of new_node1 is 0.
+    PubKey1 = get_public_key(new_node1, Cfg),
+    Balance1 = get_balance(new_node1, PubKey1, Cfg),
+    ?assertEqual(0, Balance1),
+    ct:log("Balance of node ~p with public key ~p is ~p", [new_node1, PubKey1, Balance1]),
+    %% Sync the chain on new_node2 from new_node1.
+    start_node(new_node2, Cfg),
+    wait_for_height_syncing(TopHeight, [new_node2], {{45000, ms}, {200, blocks}}, Cfg),
+    B2 = get_block_by_height(new_node2, TopHeight, Cfg),
+    ?assertEqual(TopHash, maps:get(hash, B2)),
+    %% Make sure balance of new_node2 is 0.
+    PubKey2 = get_public_key(new_node2, Cfg),
+    Balance2 = get_balance(new_node2, PubKey2, Cfg),
+    ?assertEqual(0, Balance2),
+    ct:log("Balance of node ~p with public key ~p is ~p", [new_node2, PubKey2, Balance2]),
+    %% Start mining on new_node2.
+    ok = mock_pow_on_node(new_node1, Cfg),
```

Comment:
This is unnecessary.
Please consider that as of v0.11.0 running node started from DB of v0.10.1 is not supported so this new_node1 shall be used only for making other node sync.